internal/graphql/model/operator_helper.go

```diff
@@ -172,6 +176,7 @@ func SubscribeToBroker(provider models.Provider, mesheryKubeClient *mesherykube.
 		if err == nil && broker.Status.Endpoint.External != "" {
 			break
```

Comment:
@Revolyssup the error handling here seems like it needs improvement.

Reply:
I think it is fine. It spins in a loop with a timeout until we can get the broker without any errors AND it has been assigned an External IP. This logic seems fine to me. Can you point out what part of it needs improvement?